modules/contact-form/grunion-contact-form.php

```diff
@@ -3206,8 +3222,13 @@ function render() {
 		$field_label       = $this->get_attribute( 'label' );
 		$field_required    = $this->get_attribute( 'required' );
 		$field_placeholder = $this->get_attribute( 'placeholder' );
+		$field_width       = $this->get_attribute( 'width' );
 		$class             = 'date' === $field_type ? 'jp-contact-form-date' : $this->get_attribute( 'class' );
 
+		if ( ! empty( $field_width ) ) {
+			$class .= 'grunion-field-width-' . $field_width;
```

Comment:
I haven't followed the output chain but I assume this is being properly escaped?

Reply:
Yes, the entire string of classes is escaped at the point it is output with `esc_attr()`.